app/src/main/java/com/globant/counter/mvp/view/ActivityView.kt

```diff
@@ -9,16 +9,8 @@ import java.lang.ref.WeakReference
 open class ActivityView(activity: Activity) {
     private val activityRef: WeakReference<Activity> = WeakReference(activity)
 
-    val activity: Activity?
+    open  val activity: Activity?
```

Comment:
You don't need to make this open. `open` keyword is only used if you need to override this on subclasses
```suggestion
    val activity: Activity?
```